src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1584,16 +1585,18 @@ public static void info(final String info) {
 	 */
 	@SuppressWarnings("null")
 	public static void warning(final String warning) {
-		SkriptLogger.log(Level.WARNING, warning);
+		if (!Annotation.isAnnotationPresent("suppress warnings"))
```

Comment:
It seems like we might want to have some sort of Skript-annotations enum, similar to Experiments. It might just even hold SkriptPatterns or something

Reply:
Oh sure, this was just included as a demo, I think we now have a proper way of suppressing warnings anyway.

Reply:
I'm removing this, I think it's better as a follow-up.